ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -50,7 +54,7 @@ No prerequisites to fulfill.
 
```

Comment:
Select the end place... (konsistent zu Schritt 5)

Reply:
Done